### DISCUSSION TITLE
Recurring tasks

### DISCUSSION THREAD

**Discussion by isle9** [mgmeyers/obsidian-kanban#189] [Ideas & Feature Requests]
If you incorporated recurring tasks, I would immediately switch from Tasks to this plugin for my task management. 😅

**Comment by paulrudy** [comment]
I agree! Is there a possibility recurring tasks might make it on the roadmap?

Details that would be extra useful:
- permit same-day recurrences (minutes, hours, etc.)
- option for recurrence to trigger once task completed (e.g., task should recur one week after it was last marked completed)

**Comment by skebi69** [comment]
I definitely +1 this. I use Trello and would happily switch to this for many of my boards.

**Comment by ShchekodinNA** [comment]
I with that boys! It would be a really neat feature)

**Comment by daniel-gomez-mng** [comment]
It would be incredible to have this feature!
